mythtv/programs/mythfrontend/mythfexml.cpp

```diff
@@ -35,8 +36,7 @@ MythFEXML::MythFEXML( UPnpDevice *pDevice , const QString &sSharePath)
   : Eventing( "MythFEXML", "MYTHTV_Event", sSharePath)
 {
 
-    QString sUPnpDescPath =
-        UPnp::GetConfiguration()->GetValue( "UPnP/DescXmlPath", m_sSharePath );
+    QString sUPnpDescPath = GetMythDB()->GetSetting("UPnP/DescXmlPath", m_sSharePath);
```

Comment:
Not a DB setting

Reply:
Why wouldn't this be a DB setting like [mythtv/programs/mythbackend/mediaserver.cpp](https://github.com/MythTV/mythtv/pull/435/commits/2884f2d395aebf0b6192d70359fd7c3828df875b#diff-73a85b9f05d8f11a6e3c8cbcfc455d61149c46b6b1c8cf1bf2daec408149e597R99)?

Reply:
The settings that are of relevance should be migrated to DB configuration items, unless they are required to gain access to get the config items out of the DB. This involves a read config.xml, and save to DB cycle rather than just changing the code to request everything out of the DB.